### ISSUE TITLE
Have ImageRenditionField return ImageRendition alt property 

### ISSUE THREAD

**Issue by mail-liam** [wagtail/wagtail#5816]
### Is your proposal related to a problem?

The ImageRenditionField does not return the alt attribute in its [to_representation](https://github.com/wagtail/wagtail/blob/70da204eb1e8b2083e9a92af05ca895c2defa5dd/wagtail/images/api/fields.py#L31). Alt tags are useful for accessibility, and this feels like something that should be included in the "out of the box" solution Wagtail provides.

### Describe the solution you'd like

Ideally, I think ImageRenditionField could just return the [attrs_dict](https://github.com/wagtail/wagtail/blob/70da204eb1e8b2083e9a92af05ca895c2defa5dd/wagtail/images/models.py#L500) property of the AbstractRendition class. However, these have slightly differing keys - "src" vs "url", so that would be a breaking change.

### Describe alternatives you've considered

A simpler alternative would be adding the "alt" key to the OrderedDict constructed by ImageRenditionField.

**Comment by mail-liam** [comment]
I've had a play with this changes and can open up a PR on this if it is accepted. 😄
